test/Dialect/FIRRTL/specialize-layers-errors.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: circt-opt --pass-pipeline='builtin.module(firrtl.circuit(firrtl-specialize-layers))' -split-input-file -verify-diagnostics %s
+
+// expected-error @below {{unknown layer @A}}
+firrtl.circuit "Test" attributes {
+  firrtl.enable_layers = [@A]
+} {
+  firrtl.module @Test() {}
+}
+
+// -----
+
+// expected-error @below {{unknown layer @A}}
+firrtl.circuit "Test" attributes {
+  firrtl.disable_layers = [@A]
+} {
+  firrtl.module @Test() {}
+}
+
```

Comment:
Duplicate test?

Reply:
Difference is testing `disable_layers` and `enable_layers`